File: app/src/main/java/org/metabrainz/mobile/api/data/search/entity/LabelInfo.java
package org.metabrainz.mobile.api.data.search.entity;

import com.google.gson.annotations.SerializedName;

public class LabelInfo {
    private Label label;
    @SerializedName("catalog-number")
    private String catalogNumber;

    public Label getLabel() {
        return label;
    }

    public void setLabel(Label label) {
        this.label = label;
    }

    public String getCatalogNumber() {
        return catalogNumber;
    }

    public void setCatalogNumber(String catalogNumber) {
        this.catalogNumber = catalogNumber;
    }


}

File: app/src/main/java/org/metabrainz/android/data/sources/api/entities/LabelInfo.kt
package org.metabrainz.android.data.sources.api.entities

import com.google.gson.annotations.SerializedName
import org.metabrainz.android.data.sources.api.entities.mbentity.Label

class LabelInfo {
    var label: Label? = null

    @SerializedName("catalog-number")
    var catalogNumber: String? = null
}